IM_calculation/scripts/calculate_ims.py

```diff
@@ -10,14 +10,27 @@
 
 import argparse
 import os
+import pandas as pd
+import glob
 
 import IM_calculation.IM.im_calculation as calc
+from IM_calculation.Advanced_IM import advanced_IM_factory
 from qcore import utils
 from qcore import constants
 
 
 def load_args():
-    parser = argparse.ArgumentParser()
+    parent_parser = argparse.ArgumentParser(add_help=False)
+    parent_parser.add_argument(
+        "--advanced_im_config",
+        default=advanced_IM_factory.CONFIG_FILE_NAME,
+        help="Path to the advanced IM_config file",
+    )
+
+    parent_args = parent_parser.parse_known_args()
+
+    parser = argparse.ArgumentParser(parents=[parent_parser], add_help=True)
+
```

Comment:
Yes it's way cool.

It needs a separate section because it populates options based on the contents of the config.

Reply:
this was done before i started working on this branch.
but the design decision is mostly to be that since adv_im and base_im are distinctively different and do not interfere with each other, and if the base_im start become more complicated, having advanced_im_config as a parent parser means it will not be affected and alway be included if required.
@jasonmotha should have more idea about it, as he wrote the 1st version of this code.

Reply:
Ah, I see how it works.
I'm not too keen on the way it works and would rather see adv_im be a sub command but won't argue to change it